pyenv-win/bin/pyenv.bat

```diff
@@ -59,17 +55,16 @@ if exist "%exe%.bat" (
   exit /b
 )
 
-
-
 rem ====================================================================================
 :run
 
 rem update PATH to active version
-set "path=%bindir%;%bindir%\Scripts;%path%"
+if exist %bindir% set "path=%bindir%;%bindir%\Scripts;%path%"
```

Comment:
When no version set in pyenv, `bindir` dir name will be `pyenv` (cf. https://github.com/pyenv-win/pyenv-win/pull/140#discussion_r619629552)